socorro/external/boto/crashstorage.py

```diff
@@ -70,6 +69,12 @@ class BotoCrashStorage(CrashStorageBase):
         default='.dump',
         reference_value_from='resource.boto',
     )
+    required_config.add_option(
+        'json_object_hook',
+        default='configman.dotdict.DotDict',
+        from_string_converter=class_converter,
+        reference_value_from='resource.boto',
```

Comment:
I don't think that line should be there. This option is not strictly related to ``boto`` as a resource, it's related to this CrashStorage class.